From 88f4d40c9dc24e3127bfdcf88f71f07c2c122218 Mon Sep 17 00:00:00 2001
From: Kaixin Liu <liukaixin1@h-partners.com>
Date: Fri, 7 Aug 2026 09:09:34 +0800
Subject: [PATCH 10/11] update compile.sh

---
 compile.sh | 19 +++----------------
 1 file changed, 3 insertions(+), 16 deletions(-)

diff --git a/compile.sh b/compile.sh
index cec41977..48994d63 100755
--- a/compile.sh
+++ b/compile.sh
@@ -1,21 +1,8 @@
 # CFLAGS="-O3 -march=armv8.6-a+sve+i8mm+fp16+f32mm -fno-unroll-loops -fomit-frame-pointer -Wno-incompatible-pointer-types" CXXFLAGS="-O3 -march=armv8.6-a+sve+i8mm+fp16+f32mm -fno-unroll-loops -fomit-frame-pointer" cmake -B build-packA-1099bd .  -DCMAKE_BUILD_TYPE=Release -DLLAMA_CURL=OFF -DBUILD_SHARED_LIBS=OFF 
 # CFLAGS="-O3 -fno-unroll-loops -fomit-frame-pointer -Wno-incompatible-pointer-types" CXXFLAGS="-O3 -fno-unroll-loops -fomit-frame-pointer" cmake -B build-outer-5893 .  -DCMAKE_BUILD_TYPE=Debug -DLLAMA_CURL=OFF -DBUILD_SHARED_LIBS=OFF
-case $1 in
-  build-fused-sdpa)
-    _FLAGS="-O3 -funroll-loops -DNDEBUG"
-    env CFLAGS="$_FLAGS" CXXFLAGS="$_FLAGS" \
-      cmake -DCMAKE_BUILD_TYPE=Release \
-            -B $1 -GNinja \
-            -DLLAMA_CURL=OFF -DGGML_CCACHE=OFF \
-            -DGGML_CPU_ARM_ARCH="armv8.6-a+dotprod+i8mm+sve" \
-            -DGGML_NATIVE=OFF
-    ;;
-  *)
-    _FLAGS="-O3 -funroll-loops"
-    env CFLAGS="$_FLAGS" CXXFLAGS="$_FLAGS" \
-      cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+_FLAGS="-O3 -funroll-loops"
+env CFLAGS="$_FLAGS" CXXFLAGS="$_FLAGS" \
+cmake -DCMAKE_BUILD_TYPE=Release \
             -B $1 -GNinja \
             -DLLAMA_CURL=OFF -DGGML_CCACHE=OFF
-    ;;
-esac
 cmake --build $1 --config release --target ggml-cpu llama-embedding llama-bench llama-server -j 20 
-- 
2.43.0

